next.clayui.com/content/docs/components/drop-down.mdx

```diff
@@ -36,16 +39,81 @@ import {DropDown} from '../../../src/components/clay/DropDown';
 	Check the <a href="https://liferay.design/lexicon">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/dropdowns/">Dropdown Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
 </div>
 
-DropDown is delivered for composition offering flexibility for customization.
+<div class="clay-site-alert alert alert-warning">
+	<strong class="lead">Warning</strong>
+	Before you get started with this doc, read about <a href="/docs/foundations/composing.html" target="_blank">Clay's fundamentals of composition</a> to move on. The information below is assuming you have read about <a href="/docs/foundations/composing.html" target="_blank">Clay's compositional philosophy</a> and learned about the terms.
+</div>
+
+DropDown is delivered for composition offering flexibility for customization. DropDown's composition gives you the space to create more complex DropDowns, such as adding a form within DropDown, and creating fast DropDowns with the high-level `ClayDropDownWithItems` component that covers most Lexicon specification use cases.
 
 <DropDown />
 
+## Composing
+
 You may want to use only the content of DropDown for other cases, such as mounting an `Autocomplete`, so use `DropDown.Menu` instead of `DropDown` which gives you the flexibility to only control content without a trigger.
 
 <div class="clay-site-alert alert alert-warning">
 	<code class="language-text">DropDown</code> consists of a <code class="language-text">DropDown.Menu</code> that controls the trigger and uses it as the basis for aligning DropDown content.
 </div>
 
+#### Controlling Trigger
+
+You may want to create a trigger that is not necessarily in the same tree as DropDown, due to HTML markup issues, for these cases you can use `<ClayDropDown.Menu />`.
+
+Using `<ClayDropDown.Menu />` allows you to better control the state of DropDown but you will have to deal with focus management, control expands and other details.
+
+```jsx
+import ClayDropDown from '@clayui/drop-down';
+import React, {useState, useRef} from 'react';
+
+const Menu = ({
+    children,
+    hasLeftSymbols,
+    hasRightSymbols,
+}) => {
+    const triggerElementRef = useRef(null);
+    const [expand, setExpand] = useState(false);
+    const menuElementRef = useRef(null);
+
+    const handleExpand = event => {
+        // This is not ideal for allowing you to have more than
+        // one trigger for the same content but it simulates the
+        // advantages of controlling `DropDown.Menu`.
+        triggerElementRef.current = event.target;
+
+        setExpand(!expand);
+    };
+
+    return (
+        <div {...}>
```

Comment:
Hmm the intention was more to show that you would have more information on div, classNames etc... since I had written that you can control DropDown trigger using `DropDown.Menu` when you have tagging issues. So that was more to symbolize.

Reply:
I don't think users will think they _can't_ use other args on those elements, by leaving it out I think we communicate the same thing. Otherwise, if a user comes across and example without the `{...}` they may be unsure how to reconcile an example with and one without. So IMO I think its best to leave those out.

Reply:
Yep, I think you're right.